src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={cn({
+              'is-active': selectedTab.title === tab.title,
+            })}
+            data-cy="Tab"
+            key={tab.id}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={
+                () => {
+                  if (selectedTab.title !== tab.title) {
+                    onTabSelected(tab);
```

Comment:
maybe better to use with logic operator also just compare only `selectedTab` and `tab` for decreasing your code like: 
```js
selectedTab !== tab && onTabSelected(tab)
```
friend's hint

Reply:
you can do like Oleksandr said or move this function to a separate handler to improve readability and maintainability